src/VisualStudio/IntegrationTest/IntegrationTests/LanguageServerProtocol/LspGoToDefinition.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Microsoft.VisualStudio.IntegrationTest.Utilities;
+using Microsoft.VisualStudio.IntegrationTest.Utilities.Common;
+using Roslyn.Test.Utilities;
+using Roslyn.VisualStudio.IntegrationTests.CSharp;
+using Xunit;
+
+namespace Roslyn.VisualStudio.IntegrationTests.LanguageServerProtocol
+{
+    /// <summary>
+    /// A set of LSP specific goto definition tests.
+    /// These tests test behavior that only applies to the LSP version of goto definition.
+    /// </summary>
+    [Collection(nameof(SharedIntegrationHostFixture))]
+    public class LspGoToDefinition : AbstractEditorTest
+    {
+        protected override string LanguageName => LanguageNames.CSharp;
+
+        public LspGoToDefinition(VisualStudioInstanceFactory instanceFactory)
+            : base(instanceFactory, nameof(LspGoToDefinition))
+        {
+        }
+
+        /// <summary>
+        /// This test corresponds to the <see cref="CSharpGoToDefinition.GoToDefinitionWithMultipleResults"/> test.
+        /// The non-LSP version will not work due to feature parity issues with the multiple results window, mainly
+        /// different naming and grouping functionality.
```

Comment:
It'd be nice if this had links to the tracking items that we think are blocking.